build/buildrelease.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$1" = "" ]]
+then
+  echo "Usage: buildrelease.sh version"
+  echo "e.g. buildrelease.sh 2.0.0"
+  echo "or buildrelease.sh 2.0.0-rc1"
+  echo "It is expected that a git tag will already exist"
+  exit 1
+fi
+
+declare -r VERSION=$1
+declare -r SUFFIX=$(echo $VERSION | cut -s -d- -f2)
+declare -r BUILD_FLAG=$([ "$SUFFIX" != "" ] && echo "--version-suffix $SUFFIX")
```

Comment:
Definitely assuming bash. Will have to find a resource to understand how that works...